cset-workflow/app/install_website_skeleton/file/html/static/script.js

```diff
@@ -71,58 +71,70 @@ function construct_sidebar_from_data(data) {
   // Button icons.
   const icons = { left: "◧", full: "▣", right: "◨" };
 
-  for (category in data) {
+  for (const category in data) {
     // Details element for category.
     const details = document.createElement("details");
-    details.open = true;
 
-    // Title for category (summary element).
+    // Title for category and case_date (summary element).
```

Comment:
```suggestion
    // Title for category (summary element).
```
Revert, as it is the next one that is the case_data title.